docs/DeveloperGuide.md

```diff
@@ -234,6 +234,15 @@ due to a limitation of PlantUML, the lifeline reaches the end of diagram.
 * **Alternative 2**: Check duplicated patient by patient's `name`.
     * Pros: Easier to view as patient's `name` will be easier to be remembered.
     * Cons: Patients may have the same name.
+
+#### Aspect of Handling Multiple Inputs for Same Field 
+* **Alternative 1**: Adds `,` to the specific field
+  * Pros: Ensures that there are no duplicate prefixes in the input command.
+  * Cons: Display may be messy if inputs are long with more than 1 comma. 
+    <br></br>
+* **Alternative 2 (current choice)**: Allow duplicate prefixes for certain fields 
+  * Pros: Easier to view as listings will be shown without commas, provide clearer view to user.
+  * Cons: User may find it confused if same prefix not being entered continuously.
```

Comment:
Could consider moving current choice to alternative 1 to standardise across DG

Reply:
great suggestion!